packages/x-archetype-utils/package.json

```
@@ -42,8 +42,9 @@
     "@types/jest": "~27.0.3",
     "@types/node": "~16.4.12",
     "@vue/test-utils": "~1.0.3",
-    "concurrently": "~7.6.0",
     "jest": "~27.3.1",
+    "rollup": "~2.79.1",
+    "rollup-plugin-copy": "~3.4.0",
```

add these dependencies:

```suggestion
    "rollup-plugin-copy": "~3.4.0",
    "rollup-plugin-delete": "~2.0.0",
    "rollup-plugin-typescript2": "~0.34.1",
```